server-spi-private/src/main/java/org/keycloak/models/Constants.java

```diff
@@ -73,5 +74,12 @@
 
     int DEFAULT_MAX_RESULTS = 100;
 
+    // Delimiter to be used in the configuration of authenticators (and some other components) in case that we need to save
+    // multiple values into single string
+    String CFG_DELIMITER = "##";
+
+    // Better performance to use this instead of String.split
+    Pattern CFG_DELIMITER_PATTERN = Pattern.compile(CFG_DELIMITER);
```

Comment:
`public static`

Would be better to ignore whitespace around the delimiter? `"\\s*##\\s*"`?

Reply:
Fixed, Thanks!